database/buildSchema/06_trigger_queue.sql

```diff
@@ -1,5 +1,7 @@
 -- trigger queue
 
+CREATE TYPE public.trigger as ENUM ('onApplicationCreate', 'onApplicationSubmit', 'onApplicationSave', 'onApplicationWithdrawn', 'onReviewCreate', 'onReviewSubmit', 'onReviewStart', 'onReviewAssign', 'onApprovalSubmit', 'onScheduleTime', 'Processing', 'Error');
```

Comment:
```suggestion
CREATE TYPE public.trigger as ENUM ('onApplicationCreate', 'onApplicationSubmit', 'onApplicationSave', 'onApplicationWithdraw', 'onReviewCreate', 'onReviewSubmit', 'onReviewStart', 'onReviewAssign', 'onApprovalSubmit', 'onScheduleTime', 'Processing', 'Error');
```
Tiny thing -- "Withdrawn" should probably be "Withdraw" to keep the verb tense consistent with all the rest. (We haven't used it yet, so won't affect anything)